share/themes/darkmode.conf

```diff
@@ -0,0 +1,25 @@
+#
+# GMT 6.1.0 Defaults file for darkmode theme
```

Comment:
Do we really want to have the GMT version in the theme files?

Reply:
No, removed.